src/python/pants/option/config.py

```diff
@@ -137,10 +136,7 @@ def _getinstance(self, section, option, type_, default=None):
       return default
 
     raw_value = self.get_value(section, option)
-    # We jump through some hoops here to deal with the fact that `six.string_types` is a tuple of
-    # types.
-    if (type_ == six.string_types or
-        (isinstance(type_, type) and issubclass(type_, six.string_types))):
+    if type_ == str:
```

Comment:
Can `str` be subclassed? should we use issubclass() for safety?

Reply:
Yes, `str` can be subclassed, you're right. I don't see any places we do that, but I agree that we should add back the `issubclass` check.